doc/task_breakdown.md

```diff
@@ -0,0 +1,54 @@
+# SQL Query Builder Task Breakdown 
+
+## Task 1: UI and initial structure (2 days)
+- Create a UI layout with sections: Tables, Joins, SQL Query, Columns, Filters, Sorting, Query Result.
+- Provide an interface where users can configure query parameters step by step.
+- Display the SQL query dynamically as the user selects tables, columns, filters, and other query elements.
+- Ensure that each selection updates the displayed SQL query but does not execute it yet.
```

Comment:
What sections can update SQL?
this is not possible inside this task, there is nothing before it